Modules/Loadable/Markups/MRML/vtkMRMLMarkupsCurveNode.cxx

```diff
@@ -1273,22 +1274,14 @@ void vtkMRMLMarkupsCurveNode::OnSurfaceModelTransformChanged()
     }
 
   vtkSmartPointer<vtkGeneralTransform> surfaceToWorldTransform = vtkGeneralTransform::SafeDownCast(
-    this->SurfaceToWorldTransformer->GetTransform());
+    this->SurfaceToLocalTransformer->GetTransform());
   if (!surfaceToWorldTransform)
     {
     surfaceToWorldTransform = vtkSmartPointer<vtkGeneralTransform>::New();
-    this->SurfaceToWorldTransformer->SetTransform(surfaceToWorldTransform);
+    this->SurfaceToLocalTransformer->SetTransform(surfaceToWorldTransform);
```

Comment:
We set surfaceToWorldTransform in a SurfaceToLocalTransformer. One of the names should be fixed.